### ISSUE TITLE
openshift_node_labels stopped working

### ISSUE THREAD

**Issue by sjenning** [openshift/openshift-ansible#7766]
```
[OSEv3:children]
etcd
masters
nodes

[OSEv3:vars]
ansible_ssh_user=centos
ansible_become=yes

openshift_master_identity_providers=[{'name': 'allow_all', 'login': 'true', 'challenge': 'true', 'kind': 'AllowAllPasswordIdentityProvider'}]
openshift_master_default_subdomain=apps.variantweb.net
openshift_install_examples=true
openshift_disable_check=memory_availability,disk_availability
openshift_portal_net=172.30.0.0/16
openshift_enable_service_catalog=false
template_service_broker_install=false

deployment_type=origin
openshift_additional_repos=[{'id': 'origin-devel', 'name': 'origin-devel', 'baseurl': 'https://storage.googleapis.com/origin-ci-test/logs/test_branch_origin_extended_conformance_gce/2797/artifacts/rpms', 'enabled': 1, 'gpgcheck': 0}]

[etcd]
master.lab.variantweb.net

[masters]
master.lab.variantweb.net

[nodes]
master.lab.variantweb.net
infra.lab.variantweb.net openshift_node_labels="{'region': 'infra', 'zone': 'default'}"
node[1:2].lab.variantweb.net openshift_node_labels="{'region': 'primary', 'zone': 'default'}"
```
Should be noted that the playbook shows and error:
```
INSTALLER STATUS *******************************************************************************************************************
Initialization             : Complete (0:00:13)
Health Check               : Complete (0:00:16)
etcd Install               : Complete (0:00:24)
Master Install             : Complete (0:02:02)
Master Additional Install  : Complete (0:00:25)
Node Install               : Complete (0:00:02)
Hosted Install             : Complete (0:00:34)
Web Console Install        : Complete (0:00:28)

Tuesday 03 April 2018  17:06:38 -0500 (0:00:00.015)       0:09:44.205 ********* 
=============================================================================== 
openshift_node : Start and enable node for bootstrapping ------------------------------------------------------------------ 300.46s
openshift_control_plane : Verify that the control plane is running --------------------------------------------------------- 55.61s
openshift_web_console : Verify that the web console is running ------------------------------------------------------------- 22.62s
Run health checks (install) - EL ------------------------------------------------------------------------------------------- 15.90s
Approve bootstrap nodes ----------------------------------------------------------------------------------------------------- 9.42s
openshift_hosted : Create OpenShift router ---------------------------------------------------------------------------------- 7.93s
openshift_control_plane : create service account kubeconfig with csr rights ------------------------------------------------- 5.67s
openshift_manageiq : Configure role/user permissions ------------------------------------------------------------------------ 4.71s
openshift_hosted : Create default projects ---------------------------------------------------------------------------------- 3.28s
etcd : Install etcd --------------------------------------------------------------------------------------------------------- 2.66s
openshift_hosted : Add the secret to the registry's pod service accounts ---------------------------------------------------- 2.47s
Gather Cluster facts -------------------------------------------------------------------------------------------------------- 1.96s
openshift_node : Gather node information ------------------------------------------------------------------------------------ 1.81s
openshift_manage_node : label non-master non-infra nodes compute ------------------------------------------------------------ 1.71s
openshift_hosted : Create OpenShift registry -------------------------------------------------------------------------------- 1.61s
openshift_master_certificates : Check status of master certificates --------------------------------------------------------- 1.51s
openshift_sdn : Copy templates to temp directory ---------------------------------------------------------------------------- 1.50s
openshift_manageiq : Create Admin and Image Inspector Service Account ------------------------------------------------------- 1.50s
openshift_ca : Create the master certificates if they do not already exist -------------------------------------------------- 1.50s
openshift_hosted : create the default registry service ---------------------------------------------------------------------- 1.47s


Failure summary:


  1. Hosts:    infra.lab.variantweb.net, node1.lab.variantweb.net, node2.lab.variantweb.net
     Play:     Distribute bootstrap and start nodes
     Task:     openshift_node : fail
     Message:  Node start failed.
```

After the cluster installs and after manually setting `BOOTSTRAP_CONFIG_NAME` in `/etc/sysconfig/origin-node` (see https://github.com/openshift/openshift-ansible/pull/7762), I noticed that the labels I set with `openshift_node_labels` are not applied to the nodes:
```
$ oc get nodes --show-labels
NAME                        STATUS    ROLES            AGE       VERSION             LABELS
infra.lab.variantweb.net    Ready     compute,infra    3h        v1.9.1+a0ce1bc657   beta.kubernetes.io/arch=amd64,beta.kubernetes.io/os=linux,kubernetes.io/hostname=infra.lab.variantweb.net,node-role.kubernetes.io/compute=true,node-role.kubernetes.io/infra=true
master.lab.variantweb.net   Ready     compute,master   3h        v1.9.1+a0ce1bc657   beta.kubernetes.io/arch=amd64,beta.kubernetes.io/os=linux,kubernetes.io/hostname=master.lab.variantweb.net,node-role.kubernetes.io/compute=true,node-role.kubernetes.io/master=true
node1.lab.variantweb.net    Ready     compute          3h        v1.9.1+a0ce1bc657   beta.kubernetes.io/arch=amd64,beta.kubernetes.io/os=linux,kubernetes.io/hostname=node1.lab.variantweb.net,node-role.kubernetes.io/compute=true
node2.lab.variantweb.net    Ready     compute          3h        v1.9.1+a0ce1bc657   beta.kubernetes.io/arch=amd64,beta.kubernetes.io/os=linux,kubernetes.io/hostname=node2.lab.variantweb.net,node-role.kubernetes.io/compute=true
```
@smarterclayton @sdodson

**Comment by smarterclayton** [comment]
Still investigating.  The labels task should be capturing output, not sure what is blocking it.

**Comment by smarterclayton** [comment]
I think this is fixed in the latest #7723.
